integrations/dalle/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "dalle",
```

Comment:
**chore:** rename for `@botpresshub/dalle` for coherence with other integrations. This package won't be published on NPM, it's mostly for scripting selectors in the `botpress/botpress` monorepo